src/site/workspace.scss

```diff
@@ -536,14 +536,39 @@ html {
   .example-border {
     padding: .5rem;
   }
+}
+
+.Preview__body {
+  .example-border {
+    background: none;
+    border: 2px dashed #393f44;
+    box-shadow: none;
+  }
+}
 
+// Toolbar documentation
+.toolbar-documentation {
```

Comment:
These will all be lost when we show the documentation on patternfly.org - please chat with me so we can discuss a different way to accomplish this.  If something in the preview is deficient for showing toolbar examples - we'll need to make a global change instead.